scipy/constants/_constants.py

```diff
@@ -269,19 +268,21 @@ def convert_temperature(
     array([ 233.15,  313.15])
 
     """
+    xp = array_namespace(val)
+    val = xp.asarray(val)
```

Comment:
this line is unnecessary

Reply:
Do you mind elaborating here for my understanding, do we not need this in case val is a list for example?

Reply:
right, ignore me, that seems correct